src/components/commons/repository/RepositoryCard.js

```diff
@@ -11,27 +12,39 @@ type RepositoryProps = {
 type StatProps = {
     iconClass: string,
     statCount: number,
+    tooltipMessage: String,
```

Comment:
tenta substituir aqui, creio irá resolver o erro:
```suggestion
    tooltipMessage: string,
```

Reply:
obrigado pela ajuda, estava tentando ver como olhava o erro do Travis CI

Reply:
Nada!